test/e2e/support/utils.js

```diff
@@ -70,20 +70,36 @@ export async function newPost( postType ) {
 	await visitAdmin( 'post-new.php', postType ? 'post_type=' + postType : '' );
 }
 
+export async function getWindowWidth() {
+	return await page.evaluate(
+		() => {
+			return window.innerWidth;
+		}
+	);
+}
+
 export async function newDesktopBrowserPage() {
 	global.page = await browser.newPage();
+	await setDesktopViewPort();
```

Comment:
Capitalization should be "Viewport", if even just for consistency with Puppeteer:

https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagesetviewportviewport

Reply:
Capitalization corrected, thank your for the catch.